front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -342,10 +342,12 @@ export function AgentMessage({
                 setHoveredReference: setLastHoveredReference,
               }}
             />
-            <Citations
-              activeReferences={activeReferences}
-              lastHoveredReference={lastHoveredReference}
-            />
+            <div className="overflow-x-hidden">
```

Comment:
That seems weird. Citations is supposed to handle itself according to available width? We definitely don't want to stack such classes outside/inside components?

Reply:
I don't know, but I sure prefer having this around the citations and not around the entire app